build-carthage.sh

```diff
@@ -1,9 +1,21 @@
 #!/usr/bin/env bash
 
+set -euvx
+
 FRAMEWORK_NAME="${1:-MozillaAppServices.framework.zip}"
 carthage update --platform iOS swift-protobuf
 ## When https://github.com/Carthage/Carthage/issues/2623 is fixed,
 ## carthage build --archive should work to produce a zip
-carthage build --no-skip-current --platform iOS --verbose && \
-  (cd Carthage/Build/iOS | egrep -v 'Static|Logins|FxAClient' | xargs rm -rf) && \
-  zip -r $FRAMEWORK_NAME Carthage/Build/iOS
+
+carthage build --no-skip-current --platform iOS --verbose
+
+ZIP_DIR=$(mktemp -d)
+mkdir -p $ZIP_DIR/Carthage/Build/iOS
+cp -r Carthage/Build/iOS/Static $ZIP_DIR/Carthage/Build/iOS
```

Comment:
`cp -r Carthage/Build/iOS/Static/*.framework $ZIP_DIR/Carthage/Build/iOS`
Consumers won't know to look for a `Static` dir –I certainly wouldn't anyway. There is little mention of this in Carthage docs that the type of linkage is reflected in the output directory.